addons/actions/src/containers/ActionLogger/index.js

```diff
@@ -56,21 +56,25 @@ export default class ActionLogger extends React.Component {
   }
 
   render() {
+    const { active } = this.props;
     const props = {
       actions: this.state.actions,
       onClear: () => this.clearActions(),
     };
-    return <ActionLoggerComponent {...props} />;
+    return active ? <ActionLoggerComponent {...props} /> : null;
```

Comment:
What would your solution look like?

An if statement above?

Reply:
Yeah :) 

```
if (!active) {
  return null;
}

return <ActionLoggerComponent {...props} />;
```

I do this because I can change the two conditions easier if needed and it makes it more readable imho.